### DISCUSSION TITLE
Inquiry Regarding Reverting to Free Trial Plan

### DISCUSSION THREAD

**Discussion by Doyu-Lee** [vercel/community#3652] [Help] [answered]
### Summary

Dear Support Team,

As a member of a team 'SeonBBang', we initially started with a free trial plan to explore our project in a more casual setting. During the course of our trial, we had one month where our deployments exceeded the limit of 100 times, prompting us to upgrade to the premium plan.

However, our team's current activities have slowed down significantly, and we anticipate that our deployments will remain around 10 times or less per month. With this in mind, we are considering reverting back to the free trial plan. When I attempted to cancel the current premium plan, I noticed that the provided instructions only guide us through a process that involves deleting the associated project.

Could you please provide guidance on how we can transition back to the free trial plan without losing our project? We greatly appreciate your assistance in this matter.

Thank you for your prompt attention to this inquiry.

Best regards,

Seona
SeonBBang

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/125176463/8f8b2ce8-51ba-4493-9dbd-c9419e953ac2)

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @Doyu-Lee. The free Pro trial ends after 14 days or after your team exceeds the trial usage limits, whichever happens first. Since you are already on a Pro plan, you have a couple of options:
- Continue with the existing Pro plan
- Cancel the Pro plan and delete the team
  - If your projects are for [non-commercial use](https://vercel.com/docs/concepts/limits/fair-use-policy#commercial-usage) and otherwise meet the requirements outlined in the [Fair Use Policy](https://vercel.com/docs/concepts/limits/fair-use-policy#) and plan [Limits](https://vercel.com/docs/concepts/limits/overview#) documentation, then you could transfer the projects to your Hobby account before deleting the team. In this case you would be the only person able to control the site settings, etc.
